modules/cnv-cloning-local-volume-to-another-node.adoc

```diff
@@ -0,0 +1,125 @@
++
+[source,yaml]
+----
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: <destination-pv> <1>
+  annotations:
+spec:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 10Gi <2>
+  local:
+    path: /mnt/local-storage/local/disk1 <3>
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - <node01> <4>
+  persistentVolumeReclaimPolicy: Delete
+  storageClassName: local
+  volumeMode: Filesystem
+----
+<1> The name of the PV.
+<2> The size of the PV. You must allocate enough space, or the cloning operation fails. The size must be the same as or larger than the source PVC.
+<3> The mount path on the node.
+<4> The name of the node on which you want the PV to be created.
+
+* You can identify the node on which the PV is provisioned by examining the `nodeAffinity` field of the PV configuration, in particular the `kubernetes.io/hostname` key and matching values:
++
+----
+$ oc get pv <destination-pv> -o yaml
+----
++
+The following snippet shows that the PV is on `node01`:
++
+[source,yaml]
+----
+...
+spec:
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - node01
+...
+----
+
+. Once you have created or identified the PV on the correct node, add a unique label to that PV:
++
+----
+$ oc label pv <destination-pv> node=node01
+----
+
+. Create a DataVolume manifest that references the PVC of the virtual machine you are cloning to the new node, and the `selector` label that was applied to the PV in the previous step:
++
+[source,yaml]
+----
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: DataVolume
+metadata:
+  name: <clone-datavolume> <1>
+spec:
+  source:
+    pvc:
+      name: "<source-vm-disk>" <2>
+      namespace: "<source-namespace>" <3>
+  pvc:
+    accessModes:
+      - ReadWriteOnce
+    selector:
+      matchLabels:
+        node: node01 <4>
+    resources:
+      requests:
+        storage: <10Gi> <5>
+----
+<1> The name of the new DataVolume.
+<2> The name of the source PVC. If you do not know the PVC name, you can find it in the virtual machine configuration: `spec.volumes.persistentVolumeClaim.claimName`.
+<3> The namespace where the source PVC exists.
+<4> The label applied to the PV in the previous step.
+<5> The size of the new DataVolume. You must allocate enough space, or the
+cloning operation fails. The size must be the same as or larger than the source PVC.
+
+. Apply the DataVolume manifest to initiate the cloning operation:
++
+----
+$ oc apply -f <clone-datavolume.yaml>
+----
+
+The DataVolume clones the PVC of the virtual machine into the new PV on the specific node. After the cloning operation is complete, you can edit the virtual machine configuration so that it uses the new DataVolume, or add the new DataVolume to another virtual machine. 
```

Comment:
I wonder if you should put this info (starting with "After the cloning operation...") as two bullet points at the end of the assembly instead, so that you can xref to assemblies that describe how to do these actions.

Reply:
Great idea